action.yml

```diff
@@ -208,4 +208,13 @@ runs:
             cd ${SITE_ROOT}
           fi
 
+          echo "Deleting Pantheon PR multidev environment..."
           terminus build:env:delete:pr $PANTHEON_SITE --yes
+
+          echo "Finding closed PRs to clean up GitHub environments..."
+          CLOSED_PRS=$(gh pr list --state closed --json number --jq '.[].number')
```

Comment:
This probably won't work for repos with hundreds or thousands of closed PRs where the API response paginates.

Reply:
Although, that was already part of my script that you had linked to that I copy/pasted from, so I've kept it (just in a different format based on old multidevs). It's a different, more limited implementation. I still think we shouldn't care that much about old deployments from versions of the action pre-1.x